contrib/helm/harbor/templates/registry/registry-ss.yaml

```diff
@@ -35,16 +35,18 @@ spec:
         - containerPort: 5000
         - containerPort: 5001
         volumeMounts:
+{{- if .Values.registry.objectStorage }}
```

Comment:
Should it be `if not .Values.registry.objectStorage` ? Because when .Values.registry.objectStorage is specified, no need to create the data dir.

Reply:
you're right! the condition should be inverted and the _emptyDir_ should be removed altogether since if using an object storage, there won't be `registry-data` mount.

So the volume mount would look like this : 
```yaml
        volumeMounts:
{{- if not .Values.registry.objectStorage }}
        - name: registry-data
          mountPath: /var/lib/registry
{{- end }}
```

and volume claim :
```yaml
  {{- if and (not .Values.registry.objectStorage) .Values.persistence.enabled }}
  volumeClaimTemplates:
  - metadata:
      name: "registry-data"
# yadda yadda
  {{- end -}}
```